### ISSUE TITLE
When first loading a directory with adv3Lite stored in source tree, the src/ dir is filled with .t3s and .t3o

### ISSUE THREAD

**Issue by jimbonator** [toerob/vscode-tads3tools#6]
**Describe the bug**
I've included adv3Lite in my project directory rather than using the standard location in /usr.  (I have my reasons.)  When I installed the extension and reloaded the project, my src/ directory filled with adv3Lite object and symbol files, and the final game file as well (.t3).

Note that my .t3m has options to store all object and symbol files in an obj/ directory, and writes the final .t3 to the root of the project.  (This is a hand-crafted make file, not one generated by any tools or templates.)

**To Reproduce**
The pertinent directories are:
* adv3Lite/ - contains all the advLite .t, .h, and.tl
* src/ - my game source code
* obj/ - built by t3make

My .t3m:
```
-D LANGUAGE=english
-D MESSAGESTYLE=neu
-D INSTRUCTIONS_MENU
-lib system
-lib adv3Lite/adv3Lite
-source src/game.t
```

My t3make command line:
```
t3make -d  -we -FC -cs UTF-8 -Fy obj -Fo obj -f game.t3m -o game.t3
```

**Log**
Regarding the log: Because I was worried this was going to happen every time I opened VS Code, I deleted the .t3o and .t3s files and used VS Code's "Reload Developer Window" option.  Unfortunately, I lost the original log because of this.

**Expected behavior**
I would prefer it write the symbol and object files to the directory specified in my .t3m.  (I believe they already existed there when I started the extension, from a prior build.)

**Desktop (please complete the following information):**
Ubuntu Linux (Bionic).  VS Code 1.60.0.

**Comment by jimbonator** [comment]
Ok, I need to simplify the above.  I've removed adv3Lite from my project tree, restarted VS Code, and I'm still seeing the above behavior.

**Comment by toerob** [comment]
I see. Have you tried setting the `-Fy obj` flag in the makefile as well, if it is not specified I believe this behaviour is by design of how t3make works.

**Comment by toerob** [comment]
And the extension uses the makefile to produce preprocessed text, otherwise it won’t work. I need to consider what should happen in a case like this though... e.g warning or separate directory for the preprocessing to use...

Thanks for the report by the way!

**Comment by jimbonator** [comment]
Yes, that did it.  I've tried adding various options to the .t3m in the past and t3make complained.  I realized just now that the order of the options is important.  When I moved -Fy and -Fo up towards the beginning of the file, it all worked and everything clicked into place.

I'll let you decide if you want to close this ticket, or what further work is needed, if any.  I'm satisfied.

Thanks for the great tool!
